test/test.dart

```diff
@@ -685,3 +687,40 @@ Future<void> _printPlatformInfo() async {
 
   print('Current PID $pid; OS $os, $pointerSize bit, CPU ${cpu ?? 'unknown'}');
 }
+
+Future<void> triggerClientReset(Realm realm, {bool restartSession = true}) async {
```

Comment:
do we need these methods in test.dart. Why don't we have it only in the client_reset_test.dart